packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -143,7 +143,7 @@
         margin: -11px 0px 22px 0px; // first-level lists need padding
     }
 
-    .paragraph ul:not(.perseus-widget-radio) {
+    div:not(.hint) .paragraph ul:not(.perseus-widget-radio) {
```

Comment:
**suggestion**: Also related to above comments, can this just be `div:not(.perseus-hint-renderer)`?

Reply:
Unfortunately, `perseus-hint-renderer` isn't added in some cases. I added `hint` to be added to the renderer in every case.